tests/features/class-wc-tests-ces-tracks.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Onboarding Themes Tests.
+ *
+ * @package WooCommerce\Admin\Tests\OnboardingThemes
```

Comment:
Comments need to be updated! :-)

Reply:
@mattsherman Thank you for the catch :) Updated it with https://github.com/woocommerce/woocommerce-admin/pull/5569/commits/103ff9682d7074fe0b3e25c33b738794c4d97025